warehouse/query-core/src/main/java/datawave/query/jexl/nodes/BoundedRange.java

```diff
@@ -30,7 +29,7 @@ public static String label() {
      * @return the new marker node
      * @see QueryPropertyMarker#create(JexlNode, Function)
      */
-    public static BoundedRange create(JexlNode node) {
+    public static JexlNode create(JexlNode node) {
```

Comment:
This can return a BoundedRange if we return a MARKER from `create(source, constructor)`.